lib/records/I_RecHttp.h

```diff
@@ -222,7 +224,7 @@ struct HttpProxyPort {
   typedef HttpProxyPort self; ///< Self reference type.
```

Comment:
This should be self_type or this_type to clarify what it is ...  stuff like this or self tend to mean instances. This would make the code more clear

Reply:
@dragon512 Well maybe, but lets keep it for the future refactoring tasks that will handle such cases - this one is about moving ts/Vec away in prior to std::vector.

Reply:
I agree